clients/cmd/promtail/main.go

```diff
@@ -71,6 +73,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	if config.checkSyntax {
+		if config.configFile == "" {
+			fmt.Println("Invalid config file")
+			os.Exit(0)
```

Comment:
This should return 1 instead of 0 so it can be picked up in a script.

Reply:
Thanks for the suggestion! I have resolved the issue